docs/reference/ml/anomaly-detection/aggregations.asciidoc

```diff
@@ -107,15 +123,25 @@ PUT _ml/datafeeds/datafeed-farequote
 
 <1> In this example, the aggregations have names that match the fields that they
 operate on. That is to say, the `max` aggregation is named `time` and its
-field is also `time`. The same is true for the aggregations with the names
-`airline` and `responsetime`. 
+field also needs to be `time`.
+<2> In this example, the aggregations have names that match the fields that they
```

Comment:
> <2> In this example, the aggregations have names that match the fields that they ...

IMO rather than repeating these sentences, you could just say something like this:

> <2> Likewise, the `term` aggregation is named `airline` and its field is also named `airline`.